tests/integration/app_routes/admin/admin_routes/test_templates.py

```
@@ -182,9 +181,7 @@ def test_edit_template_found(app_mock, monkeypatch):
         updated_at=None,
         source=None,
     )
-    monkeypatch.setattr(
-        "src.main_app.app_routes.admin_routes.templates.template_service.get_template", lambda id: template
-    )
+    monkeypatch.setattr("src.main_app.app_routes.admin_routes.templates.get_template", lambda id: template)
```

Ruff (0.15.12)

[error] Lambda argument `id` is shadowing a Python builtin

(A006)